lib/middleware/websocket/create_player.rb

```diff
@@ -0,0 +1,9 @@
+require 'httparty'
+
+module Websocket
+  class CreatePlayer
+    def call
+      HTTParty.post('http://localhost:3000/api/players')
```

Comment:
I dislike the idea of making HTTP calls to ourselves. Do we even need an endpoint to create players if this is our path forward?

Reply:
@jgayfer fixed.